src/db.py

```diff
@@ -93,47 +99,53 @@ def add_user(self, user: dict):
             role="user",
             use_reminder=False
         )
-
         session.add(new_user)
         session.commit()
         session.refresh(new_user)
         id = new_user.id
         session.close()
-
         if id is not None:
+            _logger.warning("User has been added successfully. STATUS_OK")
```

Comment:
This is not an exception, so
`_logger.info("User %d has been added successfully. STATUS_OK", user["tg_id"])`